tests/e2e/config/bootstrap.js

```diff
@@ -243,6 +243,20 @@ function observeNavigationRequest( req ) {
 	}
 }
 
+function requestHandlerIdea( request ) {
```

Comment:
@tofumatt could you please add jsdoc to this function?

Reply:
Also, looks like this function doesn't do anything. Do we really need it or we can remove it?

Reply:
Good catch! I commented out what it was doing because it looked like debugging left in the PR by @danielgent… then it passed and I forgot to remove this last bit 😅 

Thanks! Removed 👍🏻